src/vmm/src/vstate/vcpu/x86_64.rs

```diff
@@ -141,11 +141,18 @@ pub struct KvmVcpu {
     pub index: u8,
     /// KVM vcpu fd.
     pub fd: VcpuFd,
+    /// Vcpu peripherals, such as buses
+    pub(super) peripherals: Peripherals,
```

Comment:
Do we need a `(super)` here?

Reply:
Well, we do not need it to be accessible beyond the vcpu module. Although I'd also be fine with `pub(crate)`, just as long as we don't export it outside of the crate. We have a _ton_ of dead code that rustc doesnt report to us as dead because its marked as `pub` instead of something more restrictive, so I try to not make that problem worse.

Reply:
Ok, let's leave this for now.